ci/docker/runtime_functions.sh

```diff
@@ -774,6 +774,58 @@ build_docs() {
     popd
 }
 
+
+# Functions that run the nightly Tests:
+
+#Runs Apache RAT Check on MXNet Source for License Headers
+nightly_test_rat_check() {
+    set -ex
+    ./tests/nightly/apache_rat_license_check/license_check.sh
+}
+
+#Checks MXNet for Compilation Warnings
+nightly_test_compilation_warning() {
+    set -ex
+    export PYTHONPATH=./python/
+    ./tests/nightly/compilation_warnings/compilation_warnings.sh
+}
+
+#Checks the MXNet Installation Guide - currently checks pip, build from source and virtual env on cpu and gpu
+nightly_test_installation() {
+    set -ex
+    source ./tests/jenkins/run_test_installation_docs.sh docs/install/index.md 1 1686; ${1}
```

Comment:
Could you document these arguments? Right now, I don't understand what they would do